react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import { Header } from './Header';
+import { ImageContainer } from './ImageContainer';
+import { ImageCaption } from './ImageCaption';
+import { ImageDescription } from './ImageDescription';
+import { ButtonContainer } from './ButtonContainer';
+import './App.css';
+
+const headerText: string = 'React Image Bank';
```

Comment:
TypeScript can figure out that `headerText` is a string, because you are assigning it a value that is a string. While there's no real problem with specifying the exact type, it is common to leave off the type designation when it's obvious and just let TypeScript figure it out.
`const headerText = 'React Image Bank';`